### PR TITLE
introduction to harvester addons

### PR DESCRIPTION
**Pull request by ibrokethecloud**
PR provides a brief introduction to Harvester Addons introduced in v1.1.0.

### CODE REVIEW

File: docs/advanced/addons.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+sidebar_label: Addons
+title: ""
+---
+
+# Addons
+_Available as of v1.1.0_
+
+Beginning with v1.1.0, Harvester will make optional functionality available as Addons.
+
+One of the keys reasons for the same is to ensure that Harvester installation footprint can be kept low, while allowing users to enable/disable functionality based on their use case or requirements.
```
Comment (guangbochen) [inline docs/advanced/addons.md:12]:
```suggestion
One of the key reasons for the same is to ensure that Harvester installation footprint can be kept low while allowing users to enable/disable functionality based on their use case or requirements.
```

File: docs/advanced/addons.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Some level of customization, is allowed for each addon, which depends on the underlying addon.
```
Comment (guangbochen) [inline docs/advanced/addons.md:14]:
```suggestion
Some level of customization is allowed for each addon, which depends on the underlying addon.
```

File: docs/advanced/addons.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+v1.1.0 currently ships with two Addons:
+* pcidevices-controller
+* vm-import-controller
```
Comment (guangbochen) [inline docs/advanced/addons.md:18]: we can add links to those two pages.

**Review by vickyhella — APPROVED** [review]
LGTM!

**Review by guangbochen — APPROVED** [review]

**Review by bk201 — APPROVED** [review]
LGTM

**Comment by bk201** [comment]
> The content LGTM! I think we also need to adjust the values of `sidebar_position` for other documents under the `Advanced` section.

Let's fix it with another PR.

**Pull request merged**